staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -80,20 +80,29 @@ type fakeObjectDefaulter struct{}
 func (d *fakeObjectDefaulter) Default(in runtime.Object) {}
 
 type TestFieldManager struct {
-	fieldManager fieldmanager.Manager
-	emptyObj     runtime.Object
-	liveObj      runtime.Object
+	fieldManager    fieldmanager.Manager
+	typeConverter   internal.TypeConverter
+	objectConverter runtime.ObjectConvertor
+	groupVersion    schema.GroupVersion
+	emptyObj        runtime.Object
+	liveObj         runtime.Object
+	userAgent       string
```

Comment:
I don't think that belongs here. You would want to be able to test with different userAgent within the same test and see what happens.

Reply:
Updated